diff --git a/PKGBUILD b/PKGBUILD
index 36b5e06..eaed0fd 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -45,6 +45,7 @@ build() {
 	    --without-gsettings
 	    --with-native-compilation
 	    --with-x-toolkit=gtk3
+	    --without-toolkit-scroll-bars
 	    --without-xaw3d
 	    --without-gpm
 	    --without-compress-install
@@ -52,7 +53,7 @@ build() {
 	)
 	cd "$srcdir/$pkgname"/build
 	../configure "${_confflags[@]}"
-	make NATIVE_FULL_AOT=1
+	make NATIVE_FULL_AOT=1 -j6
 	make html
 	make pdf
 }
